cxplat/src/cxplat_winkernel/memory_winkernel.c

```diff
@@ -7,12 +7,11 @@
 __drv_allocatesMem(Mem) _Must_inspect_result_ _Ret_writes_maybenull_(size) void* cxplat_allocate_with_tag(
```

Comment:
Why have one function with a flag, requiring that extra branch for every caller? I'd expect most callers to always pick one or the other, so if you had two functions, you'd never need a branch.

Reply:
Fair point, though once #90 is done it will be moot as compiler optimization will remove the unused branch.
Still, probably good to split in the future, the code has been this way in ebpf-for-windows and just moved here, but can clean up going forward.

Reply:
I agree that the compiler should do a good job and optimize it away, but I never have much faith in compiler optimizations. 😄